src/context.ts

```diff
@@ -54,13 +56,18 @@ export class ContextManager implements IContext {
 
     return userId instanceof Array ? userId[0] : userId;
   }
+
+  get apiId(): string {
```

Comment:
curious why you don't just set this in the constructor?

Reply:
I was going to mock it in tests similar to `userId` but somewhere along the line I decided not to and optional chain it but it looked consistent with other header properties, so I left it alone.